internal/rbd/nodeserver.go

```diff
@@ -101,7 +101,7 @@ var (
 	xfsHasReflink = xfsReflinkUnset
 
 	mkfsDefaultArgs = map[string][]string{
-		"ext4": {"-m0", "-Enodiscard,lazy_itable_init=1,lazy_journal_init=1"},
+		"ext4": {"-m0", "-Enodiscard,assume_storage_prezeroed=1"},
```

Comment:
make sure that you only do this when `mkfs.ext4` supports the option, you'll need to add some form of detection, ideally with something like `sync.Once`.

Reply:
> `assume_storage_prezeroed` option became available in e2fsprogs 1.47.0 last year, so it should probably be "discovered" similar to `xfsSupportsReflink()`.

My suggestion to do it similar to `xfsSupportsReflink()` isn't viable because `mkfs.ext4` doesn't include extended options in its help output, but trying with `assume_storage_prezeroed=1` and falling back to `lazy_itable_init=1,lazy_journal_init=1` upon parsing out `Bad option(s) specified: assume_storage_prezeroed` error should work.

Reply:
I and Madhu had a discussion on this. Since this flag is part of e2fsprogs >= 1.47 and we have control over which version of it we bundle in the container, do we really need this check?

Sidenote: Detection is possible by passing the semver of e2fsprogs. It is standard flag so it must be there?

Reply:
The check for the version isn't always reliable in the face of downstream backports.